inz-demo/internal-affairs/description.xml

```diff
@@ -0,0 +1,15 @@
+<Issuer version="4">
+    <ID>internal-affairs</ID>
+    <Name>
+        <en>Demo Internal Affairs Issuer</en>
+        <nl>Demo Internal Affairs Issuer</nl>
+    </Name>
+    <ShortName>
+        <en>Demo Internal Affairs</en>
```

Comment:
could be useful to start to line up on the data standards for name, address and a few other things. NZ currently has 3 approved data content standards which would be useful for our PoCs. See https://www.data.govt.nz/manage-data/data-content-standardisation/register-of-government-data-content-requirements/

Reply:
@camfindlay Sure! I changed to use the standard names. We can improve later creating a full credential with all pieces of the data content standard, but doing now would require lots of changes in all our PoC artefacts. I added the standard document to our library to use it whenever possible.